test/CMakeLists.txt

```diff
@@ -31,64 +37,65 @@ set( CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib )
 
 #  ========================== Coverity Analysis Configuration =======================================
 
-# Coverity can run off the unit test targets, but this avoids calling unnecessary
-# unit testing tools like gcov and CMock.
+if( COV_ANALYSIS )
```

Comment:
Can you help to describe when to use `COV_ANALYSIS` and `UNITTEST` variables?
Do we also update the [coverity README](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/blob/main/tools/coverity/README.md) to use `COV_ANALYSIS`?

Reply:
Updated! Nice catch, thanks!